arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -686,9 +683,6 @@ Result DatabaseFeature::createDatabase(CreateDatabaseInfo&& info, TRI_vocbase_t*
     }
 
     if (!engine->inRecovery()) {
-      // starts compactor etc.
-      engine->recoveryDone(*vocbase);
```

Comment:
I bet that may break ArangoSearch recovery logic

Reply:
I don't think so. `recoveryDone(vocbase)` was a no-op in all storage engines but MMFiles.